tests/test_capsule/test_capsule_serializers.py

```diff
@@ -39,26 +40,29 @@ def test_activated_capsule_serialization(alices_keys, bobs_keys):
     delegating_pubkey = delegating_privkey.get_pubkey()
     signer_alice = Signer(signing_privkey)
 
-    receiving_privkey, receiving_pubkey = bobs_keys
+    params = delegating_privkey.params
```

Comment:
This is an interesting paradigm where each key and capsule can have its own params.

This might be a great pattern in the future, but we should be careful with it for now.

Reply:
Agreed on both points 100%.  Interesting but be careful.